python/test/unit/language/test_core.py

```diff
@@ -2014,7 +2018,10 @@ def test_scan_layouts(M, N, src_layout, axis, device):
     with tempfile.NamedTemporaryFile(mode='w', suffix='.ttgir') as f:
         f.write(ir)
         f.flush()
-        kernel = triton.compile(f.name)
+        capability = 0
+        if torch.cuda.is_available():
+            capability = torch.cuda.get_device_capability()
+        kernel = triton.compile(f.name, target=(device, capability))
```

Comment:
Could we have something like:
```
kernel = triton.compile(f.name, target=get_current_target())
```
?

Reply:
Actually we don't need to specify the target anymore, as it is now default to `get_current_target()`, thanks to your earlier changes :)